test/e2e/predictor/test_lightgbm.py

```diff
@@ -139,3 +139,40 @@ def test_lightgbm_v2_runtime_kserve():
         3.0601420299625077e-06]
 
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+
+
+@pytest.mark.fast
+def test_lightgbm_v2_kserve():
+    service_name = "isvc-lightgbm-v2-kserve"
+
+    predictor = V1beta1PredictorSpec(
```

Comment:
do u need to set v2 here? how is this different from the method above `test_lightgbm_v2_runtime_kserve` ?

Reply:
should `protocol_version="v2",` be in spec?

Reply:
maybe `protocol_version` should be an array as kserve python serving runtime now supports both v1 and v2.

Reply:
Created https://github.com/kserve/kserve/issues/2784 to track this issue, if we specify v2 then kserve currently selects the mlserver runtime